squid_cleanup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -x
```

Comment:
mayb also let's exit for a non-zero status and also treat unset variables as errors